libcaf_net/src/ip.cpp

```diff
@@ -113,6 +146,114 @@ std::string hostname() {
   return buf;
 }
 
+#ifdef CAF_WINDOWS
+
+std::vector<ip_address> local_addresses(string_view host) {
+  using adapters_ptr = std::unique_ptr<IP_ADAPTER_ADDRESSES, void (*)(void*)>;
+  using wbuf_ptr = std::unique_ptr<IP_ADAPTER_ADDRESSES, void (*)(void*)>;
+  if (!host.empty() && host.compare(v4_any_addr) == 0)
+    return {ip_address{make_ipv4_address(0, 0, 0, 0)}};
+  if (!host.empty() && host.compare(v6_any_addr) == 0)
+    return {ip_address{}};
+  ULONG len = 0;
+  auto err = GetAdaptersAddresses(AF_UNSPEC, GAA_FLAG_INCLUDE_PREFIX, nullptr,
```

Comment:
The variable is used only once.

```suggestion
  if (GetAdaptersAddresses(AF_UNSPEC, GAA_FLAG_INCLUDE_PREFIX, nullptr, nullptr, &len) != ERROR_BUFFER_OVERFLOW)
```